_includes/devdoc/ref_p2p_networking.md

```diff
@@ -693,102 +693,9 @@ d91f4854 ........................... Epoch time: 1414012889
 {% autocrossref %}
 
 *Added in protocol version 311.*
+*Removed in protocol version 70013 and released in Bitcoin Core v0.13.0*
```

Comment:
Since `alert` removal did not get its own protocol number increase, I was not 100% sure what to use here.

The protocol number was 70012 when https://github.com/bitcoin/bitcoin/pull/7692 was merged on Mar 21, 2016. Then the protocol was bumped to 70013 a few hours later for `feefilter` in [#7542](https://github.com/bitcoin/bitcoin/pull/7542), then again to 70014 before v0.13.0 was finally released.  

70013 was the first version where the feature did not exist, so I used that version number.  Am I understanding that correctly?